zerver/models/saved_snippets.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+from django.db import models
+from django.utils.timezone import now as timezone_now
+
+from zerver.models.realms import Realm
+from zerver.models.users import UserProfile
+
+
+class SavedSnippet(models.Model):
+    MAX_TITLE_LENGTH = 30
+
+    realm = models.ForeignKey(Realm, on_delete=models.CASCADE)
+    user_profile = models.ForeignKey(UserProfile, on_delete=models.CASCADE)
+    title = models.CharField(max_length=MAX_TITLE_LENGTH)
```

Comment:
This should probably be a `TextField`; the benefit if that it's a lot less messy to change the `max_length` later.